src/components/Filters.vue

```diff
@@ -0,0 +1,254 @@
+<template>
+  <ion-menu side="end" content-id="main-content" type="overlay">
+    <ion-header>
+      <ion-toolbar>
+        <ion-title>{{ $t("Filters") }}</ion-title>
+      </ion-toolbar>
+    </ion-header>
+
+    <ion-content>
+      <ion-list>
+        <section v-if="segmentSelected === 'history'">
+          <ion-item>
+            <ion-label class="ion-text-wrap">
+              <p>
+                {{ $t("Status") }}
+              </p>
+            </ion-label>
+          </ion-item>
+          <ion-item button v-for="(filter, index) in statusFilters" :key="index" @click="applyStatusFilters(filter)">
+            <ion-icon slot="start" :ios="filter.iosIcon" :md="filter.mdIcon" />
+            <ion-label>{{ $t(filter.name) }}</ion-label>
+            <ion-checkbox slot="end" :checked="selectedStatusFilters.includes(filter.statusId)" />
+          </ion-item>
+        </section>
+
+        <ion-item>
+          <ion-label class="ion-text-wrap">
+            <p>
+              {{ $t("Category") }}
+            </p>
+          </ion-label>
+        </ion-item>
+        <ion-item button v-for="(filter, index) in categoryFilters" :key="index" @click="applyCategoryFilters(filter)">
+          <ion-icon slot="start" :ios="filter.iosIcon" :md="filter.mdIcon" />
+          <ion-label>{{ $t(filter.name) }}</ion-label>
+          <ion-checkbox slot="end" :checked="selectedCategoryFilters.includes(filter.enumTypeId)" />
+        </ion-item>
+
+        <section v-if="getPinnedJobs.length > 0">
+          <ion-item>
+            <ion-label class="ion-text-wrap">
+              <p>
+                {{ $t("Pinned") }}
+              </p>
+            </ion-label>
+          </ion-item>
+          <ion-item button v-for="(job, index) in getPinnedJobs" :key="index" @click="applyPinnedJobFilters(job)">
+            <ion-label>{{ getEnumName(job) }}</ion-label>
+            <ion-checkbox slot="end" :checked="selectedPinnedJobs.includes(job)" />
+          </ion-item>
+        </section>
+      </ion-list>
+    </ion-content>
+  </ion-menu>
+</template>
+
+<script lang="ts">
+import {
+  IonContent,
```

Comment:
Arrange the import for components alphabetically.

Reply:
Sure, sir